### PR TITLE
fix: Change frequency of remote change on change config (Close #333)

### LINKED ISSUE #333: Cant disable remote changes tracking
- VSCode Version: 1.27.1
- OS Version: Ubuntu 16.04.5
- Extension Version: 1.34.0
- System Language: English
- SVN Version: 1.10.0

## Issue ##
Setting
svn.remoteChanges.checkFrequency to 0 doesn't disable remote changes tracking
## Steps to Reproduce ##

1.Set "svn.remoteChanges.checkFrequency" to 0
2.Wait for remote changes notification at the Source Control window

### PR DESCRIPTION
**Pull request by edgardmessias**

### CODE REVIEW

File: src/repository.ts [hunk 1]
```diff
@@ -234,27 +234,46 @@ export class Repository {
...
-        this.updateRemoteChangedFiles();
-      }, 1000 * updateFreqNew);
 
-      this.disposables.push(
-        toDisposable(() => {
-          clearInterval(interval);
-        })
+    // Update the interval of frequency of remote changes
+    const checkUpdateFreq = () => {
+      const updateFreqNew = configuration.get<number>(
```
Comment (JohnstonCode) [inline src/repository.ts]: This cant be a const if it overwritten. Aslo it will only have scopt in the function so outside will have no visibility

Comment (edgardmessias) [reply src/repository.ts]: Your suggestion?

Comment (JohnstonCode) [reply src/repository.ts]: You just need to declare `updateFreqNew` outside that function

Comment (edgardmessias) [reply src/repository.ts]: Changed to attribute of Repository

**Comment by JohnstonCode** [comment]
@edgardmessias  take a look at my suggestion

**Comment by edgardmessias** [comment]
Now I understand what you wanted

**Comment by edgardmessias** [comment]
Done

**Comment by edgardmessias** [comment]
`continuous-integration/appveyor/branch` bug?

**Comment by JohnstonCode** [comment]
Yeah not sure whats causing it. No jobs are queued on appveyor

**Pull request merged**